org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/ProductExposer.scala

```diff
@@ -73,7 +79,12 @@ private[zinc] object ProductExposer {
       finder.collected
     }.foreach { path =>
       val fname = path.toFile.getName
-      Files.move(path, path.resolveSibling(fname.substring(0, fname.lastIndexOf(ProductExtension))), StandardCopyOption.REPLACE_EXISTING)
+      try {
+        Files.move(path, path.resolveSibling(fname.substring(0, fname.lastIndexOf(ProductExtension))), StandardCopyOption.REPLACE_EXISTING)
+      } catch {
+        case e: Throwable =>
+          logger.warn(s"IO error on show: ${e.getMessage} with resource ${path}")
```

Comment:
I think that you should mention in the log output that the exception is not fatal and that the operation simply could be skipped.